examples/basic/multi_agent_rl_example.py

```diff
@@ -0,0 +1,122 @@
+import gym
+import numpy as np
+import supersuit as ss
+from gym.envs.registration import EnvSpec
+from gym.vector.vector_env import VectorEnv
+from pettingzoo.butterfly import pistonball_v4
+from sequoia.settings.base import Method
+from sequoia.settings.base.setting import Setting as MARLSetting
+from sequoia.settings.rl import IncrementalRLSetting
+from stable_baselines3.ppo import CnnPolicy
+from stable_baselines3 import PPO
+
+try:
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal
+
+
+# This creates a vectorized environment that has multiple pistonball environments concatenated together
+# For testing, we set the num_vector_envs to 1 to create a single pistonball environment (vectorized across agents)
+def make_pistonball_env(
+        vector_env_type: Literal["gym", "stable_baselines3", "stable_baselines"] = "stable_baselines3",
+        num_vector_envs: int = 8
+) -> VectorEnv:
+    env = pistonball_v4.parallel_env(
+        n_pistons=20,
+        local_ratio=0,
+        time_penalty=-0.1,
+        continuous=True,
+        random_drop=True,
+        random_rotate=True,
+        ball_mass=0.75,
+        ball_friction=0.3,
+        ball_elasticity=1.5,
+        max_cycles=125,
+    )
+    env = ss.color_reduction_v0(env, mode="B")
+    env = ss.resize_v0(env, x_size=84, y_size=84)
+    env = ss.frame_stack_v1(env, 3)
+    vector_env: VectorEnv = ss.pettingzoo_env_to_vec_env_v0(env)
+    env = ss.concat_vec_envs_v0(vector_env, num_vector_envs, num_cpus=4, base_class=vector_env_type)
```

Comment:
Oh I just understood what this does. At the previous line, we have a VecEnv with num_envs of 20 (what we want).
Now we're actually multiplying the batch size by `num_vector_envs`, so a better name for the result of this here would be something like `vector_vector_env = ss.concat_vec_envs_v0(...)`!

Either remove this, or add an `if num_vector_envs > 1` to do this.

Reply:
Oh and also, set the default value for `num_vector_envs` to 1.